util/cron/test-cray-xc-gpu-arkouda.bash

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Run arkouda testing on a cray-cs with HDR IB
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-native-gpu.bash
+export CHPL_COMM=none
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="cray-xc-gpu-arkouda"
+
+# setup arkouda
+export CHPL_TEST_ARKOUDA_PERF=false
+source $CWD/common-arkouda.bash
+export ARKOUDA_NUMLOCALES=16
```

Comment:
Since we are stopping after build, I don't think this matters, but no harm in keeping it I suppose.

Reply:
I think I need some of the settings in `$CWD/common-arkouda.bash` for things like the paths to third party libraries, etc. 


The `ARKOUDA_NUMLOCALES` variable is useless though (I removed it).